Modelica/Mechanics/Translational.mo

```diff
@@ -3146,8 +3146,8 @@ distributions:
    Typical values of coefficients of friction:
 </p>
 <pre>
-      dry operation   :  <strong>mue</strong> = 0.2 .. 0.4
-      operating in oil:  <strong>mue</strong> = 0.05 .. 0.1
+      dry operation   :  <strong>mue</strong> = 0.2 ... 0.4
+      operating in oil:  <strong>mue</strong> = 0.05 ... 0.1
```

Comment:
Why not here?

Reply:
Fixed in c50a207b16525be5da129ec427f6a3c75687d173